### DISCUSSION TITLE
Strange default directory for new projects from Version Control?

### DISCUSSION THREAD

**Discussion by blueglyph** [intellij-rust/intellij-rust#10147] [Q&A] [answered]
Each time I start a new project with `File` / `New...` / `Project from Version Control`, I get a fancy directory instead of my default project directory, any idea how I can change that? Is it a bug or is it intended?

![image](https://user-images.githubusercontent.com/31706195/220073514-a66323cd-778c-4750-8b09-f37106a53944.png)

That directory doesn't even exist anymore on my disk. I tried the Ruby plugin once and uninstalled it weeks ago.

My settings:

![image](https://user-images.githubusercontent.com/31706195/220073597-cbbfacea-1ea2-4b75-b028-d7d7efdbd472.png)

I don't have this problem when creating a new project, it seems to be specific to version control (GitHub or generic URL).

## Environment

* **IntelliJ Rust plugin version:** 0.4.188.5205-223
* **Rust toolchain version:** 1.67.0 (fc594f156 2023-01-24) x86_64-pc-windows-msvc
* **IDE name and version:** IntelliJ IDEA 2022.3.2 Ultimate Edition (IU-223.8617.56)
* **Operating system:** Windows 10 10.0
* **Macro expansion:** enabled

**Comment by Undin** [comment, marked as answer]
It doesn't seem related to Rust plugin. So I'd create issue in platform [issue tracker](https://youtrack.jetbrains.com/issues/IDEA)

**Comment by blueglyph** [comment]
OK, so it looks like another bug? I'll create an issue, thanks!
